views/visualDiff.html

```diff
@@ -0,0 +1,6 @@
+{% if (data.workflowMode == 'live') and data.siteReview.review %}
+  <div class="apos-visual-diff" data-apos-visual-diff>
+    <iframe src="{{ data.report }}" width="1000" height="640"  frameborder="0"></iframe>
```

Comment:
Needs some kind of data attribute and appropriate scoping in its jquery, see earlier comment.

Reply:
Fixed dimensions here in template aren't great, give it a class and push styles that can be overridden normally.